### ISSUE TITLE
[Question] Is ZTNA supported ?

### ISSUE THREAD

**Issue by stulluk** [theinvisible/openfortigui#202]
Title says all. Does this client support ZTNA ?

**Comment by theinvisible** [comment]
No, ZTNA is not supported. openFortiGUI is a frontend for openfortivpn, which implements the Fortinet SSL-VPN tunnel protocol only, so ZTNA would have to be implemented upstream first.
